ocs_ci/framework/__init__.py

```diff
@@ -24,6 +24,7 @@ class Config:
     REPORTING: dict = field(default_factory=dict)
     RUN: dict = field(default_factory=dict)
     UPGRADE: dict = field(default_factory=dict)
+    OCP_UPGRADE: dict = field(default_factory=dict)
```

Comment:
I don't think we need extra section for OCP_UPGRADE. You can put what you need in UPGRADE section as it's general UPGRADE related section.

Reply:
ok